### PR TITLE
adds debuglater recipe

### PR DESCRIPTION
**Pull request by edublancas**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by edublancas** [comment]
@conda-forge/help-python ready for review!

File: recipes/debuglater/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - debuglater
+  commands:
+    - pip check
+    - debuglater --help
+    - dltr --help
+  requires:
+    - pip
```
Comment (awvwgk) [inline recipes/debuglater/meta.yaml:36]: Can we run the testsuite?

```suggestion
  commands:
    - pip check
    - debuglater --help
    - dltr --help
    - pytest tests/
  requires:
    - pip
    - pytest
  source_files:
    - tests/
```

Comment (edublancas) [reply recipes/debuglater/meta.yaml:36]: we run the tests on the main [repo](https://github.com/ploomber/debuglater/actions), I thought of skipping them here to keep runtime short

Comment (edublancas) [reply recipes/debuglater/meta.yaml:36]: Hi @conda-forge/staged-recipes, any feedback here? Can we proceed with the existing recipe? We're running the test suite already in our CI system, so we can skip it here and save some build time.

Comment (carterbox) [reply recipes/debuglater/meta.yaml:36]: Running full tests is not required for conda-forge packages. The primary purpose of the conda tests section is for testing that the packaging was successful. This does usually mean running some unit tests if the package is compiled, but for python packages imports, pip checks, and entry points are usually sufficient.

File: recipes/debuglater/LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+MIT License
```
Comment (carterbox) [inline recipes/debuglater/LICENSE:1]: Delete this file. It is already included in the tar from PYPI.

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/debuglater/LICENSE (hunk 2 above)
Comment (carterbox) [reply recipes/debuglater/LICENSE:1]: Once deleted, LGTM.

**Comment by edublancas** [comment]
done, license file deleted! @carterbox

**Pull request merged**